app/controllers/tasks_controller.rb

```diff
@@ -64,54 +64,9 @@ def complete
 
     task.update!(status: :completed, completed_at: Time.current)
```

Comment:
Can you make `task` a instance variable? And have a private method like `set_task`. Since `task = user.tasks.find(params[:id]` is used in other methods.